s3fs/tests/test_s3fs.py

```diff
@@ -93,6 +93,11 @@ def test_client_kwargs():
     assert s3.s3.meta.endpoint_url.startswith('http://foo')
 
 
+def test_config_kwargs():
+    s3 = S3FileSystem(config_kwargs={'signature_version': 's3v4'})
+    assert s3.connect(refresh=True).meta.config.signature_version == 's3v4'
```

Comment:
Should this not be `s3.connect(refresh=True).s3.meta.config.signature_version` - i.e., the boto3 client's meta, not the S3FileSystem's?